federation/pkg/federatedtypes/scheduling.go

```diff
@@ -57,11 +58,16 @@ type ReplicaSchedulingStatus struct {
 	AvailableReplicas    int32
 }
 
+type ReplicaScheduleState struct {
```

Comment:
Updated ReplicaSchedulingStatus to ReplicaStatus.
ReplicaScheduleState is the result of asapter specific schedule() (eg planner.plan for replicasets) function, which is then used to update objects into clusters.
ReplicaStatus contains the details of status fields from the cluster objects, which need accumulation to update the status of the federated object. 
Updated the comments for the same too.